et/configs.ts

```diff
@@ -0,0 +1,278 @@
+import { readFileSync, writeFileSync } from "fs"
+import { sep } from "path"
+import { findLastIndex, getUniqueByKey, getUniqueByKeyAsArray, upsertFields } from "app/helpers"
+import { addToSession } from "app/session"
+import { CaseEvent, ConfigSheets, Scrubbed } from "types/types"
+import { COMPOUND_KEYS } from "app/constants"
+
+let readTime: number = 0
+let englandwales: ConfigSheets
+let scotland: ConfigSheets
+
+/**
+ * Gets the parsed JSON file contents
+ * @param regionDir repo folder that contains definitions (ie, et-ccd-definitions-scotland)
+ * @param name of the JSON file (ie, CaseField)
+ * @returns parsed JSON file
+ */
+function getJson(regionDir: string, name: string) {
+  return JSON.parse(readFileSync(`${regionDir}${sep}definitions${sep}json${sep}${name}.json`).toString())
+}
+
+/** Getter for readTime */
+export function getReadTime() {
+  return readTime
+}
+
+/** Getter for englandwales */
+function getEnglandWales() {
+  return englandwales
+}
+
+/** Getter for scotland */
+function getScotland() {
+  return scotland
+}
+
+/**
+ * Mapper function for deciding where to write objects depending on CaseTypeID
+ * @param caseTypeId a caseTypeID value
+ * @returns either englandwales or scotland
+ */
+export function getConfigSheetsForCaseTypeId(caseTypeId: string) {
+  return caseTypeId.startsWith("ET_EnglandWales") ? getEnglandWales() : getScotland()
+}
+
+/**
+ * Get all defined CaseEvent IDs in englandwales and scotland configs
+ */
+export function getCaseEventIDOpts() {
```

Comment:
Inconsistent capitalisation of ID. I'd say stick to "Id".

Reply:
It's as inconsistent as CCD is ;) Went with "ID" instead as per camel casing rules (use Pascal case or camel case for acronyms more than two characters long)